spec/features/person_edit_spec.rb

```diff
@@ -40,7 +44,7 @@
     end
 
     it "does not duplicate channels on update" do
-      cj = create(:person, firstname: 'CJ',  department: 'Police' )
+      cj = create(:person, firstname: 'CJ', lastname: 'test',  department_id: Department.where(name: 'Police')&.first&.id )
```

Comment:
FYI, `Department.where(...)` returns an ActiveRecord::Relation, so you don't need the "sad operator" in `&.first`.  Just `.first` will always work.